### PR TITLE
583 search block update

### LINKED ISSUE #583: Search block update - Frontend
Updates needed to support calls and responses from OpenSearch instead of existing SearchStax index.

Adapt existing `Search` block of Mack to Volvotrucks.us

### PR DESCRIPTION
**Pull request by jonatan-lledo-netcentric**
Fix #583 

Test URLs:
- Before: https://main--vg-volvotrucks-us--hlxsites.hlx.page/drafts/jlledo/search-demo
- After: https://583-search-block-update--vg-volvotrucks-us--hlxsites.hlx.page/drafts/jlledo/search-demo

### search example:
http://583-search-block-update--vg-volvotrucks-us--hlxsites.hlx.page/drafts/jlledo/search-demo?q=Vanguard+Truck&start=25&tags=vnl&sort=LAST_MODIFIED_DESC

### CODE REVIEW

File: blocks/search/search-api.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+const SEARCH_URLS = {
```
[the remark this answers is not in the document]
Comment (jonatan-lledo-netcentric) [reply blocks/search/search-api.js:1]: added to constants.js

**Review by Lakshmishri — CHANGES_REQUESTED** [review]
Screenshot 2024-03-18 at 14 21 28

In mobile view i see filter not seen because of color

File: blocks/search/search.css [hunk 2]
```diff
@@ -0,0 +1,2153 @@
...
+  width: 28px;
+  display: block;
+  background-size: 100%;
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.input-container-custom .sf-header-searchstudio-js .sf-form span {
+  position: absolute;
+  width: 75px;
```
[the remark this answers is not in the document]
Comment (jonatan-lledo-netcentric) [reply blocks/search/search.css:1289]: Fixed the icon to be centered on a square button

**Comment by jonatan-lledo-netcentric** [comment]
I fixed all CSS variables imported from the MT repo. Those variables doesn't exist in the VT repo and that's why the button `Filter By` was blank

**Review by manuel-vara — APPROVED** [review]

**Pull request merged**